articles/events_executor.md

```diff
@@ -53,33 +53,36 @@ The `EventsExecutor` is characterized by two main components, i.e. the `EventsQu
 
 ### Brief Overview
 
-In order to execute generic ROS 2 entities (i.e. everything except timers), the `EventsExecutor` relies on an event queue.
+In order to execute generic ROS 2 entities, the `EventsExecutor` relies on an event queue.
 Whenever a ROS 2 entity that is associated to the executor has work to do, it will push an event into the executor's queue.
 Then the executor can process these events in a FIFO manner, without need for expensive entities look-ups.
 Processing an event results in different operations depending on the entity that generated it.
 
-Timers are executed in a separate task by a timers manager, where they are kept in a priority queue sorted according to their expiration time. The task then has to monitor only the first element of the queue and can execute its callback as soon as it expires.
+Timers are monitored in a separate task by a timers manager, where they are kept in a priority queue sorted according to their expiration time. The task then has to monitor only the first element of the queue and can execute its callback as soon as it expires. The timers manager can also push timer events into the queue, instead of executing the timer.
```

Comment:
General comment: go to a new line after a `.`

Reply:
Done in b73651f